### ISSUE TITLE
Why does scaleFactor depend on rowPixelSpacing and columnPixelSpacing

### ISSUE THREAD

**Issue by wwwillchen** [cornerstonejs/cornerstone#384]
https://github.com/cornerstonejs/cornerstone/blob/12da9ed267499345ef192d9e5f214cc79122ffc3/src/internal/getImageFitScale.js#L30-L38

My understanding is that rowPixelSpacing and columnPixelSpacing describes the ratio between real-world measurement to image pixels.

If so, then scaleFactor which is used to determine how zoomed out an image should be, shouldn't depend at all on rowPixelSpacing and columnPixelSpacing.

Instead it should be:

```js
  const verticalScale = canvas.height / image.rows;	
  const horizontalScale = canvas.width / image.columns;	
  const scale = Math.min(horizontalScale, verticalScale);	
``` 

In other words, I believe https://github.com/cornerstonejs/cornerstone/pull/255/files#diff-6b808a843b9fc83adf3c1526a92e80b2R49 may have introduced a regression.

I definitely might be mis-understanding something so any clarification would be helpful.

Thank you.

**Comment by dannyrb** [comment]
Hmm... Here are the PR comments:

> This PR incorporates:
>1. The fix implemented by @luyixin in PR-193 when an image with irregular size (row/col pixel spacing are not the same) is fit to screen.
> 2. Update the "resize" example to allow setting the image row and column pixel spacing and applied when zooming or fitting to the screen

- @Zaid-Safadi 

Which seems to directly address this bug report: https://github.com/cornerstonejs/cornerstone/issues/110

That has a better description of why the change was introduced and the problem it resolves. If things still aren't clear, please let me know.

**Comment by wwwillchen** [comment]
Thanks for the response. I see - I think was thrown off by the comment in: https://github.com/cornerstonejs/cornerstone/blob/12da9ed267499345ef192d9e5f214cc79122ffc3/src/enabledElements.js#L74-L75

After re-reading the bug, I realize now that row/col spacing is actually used for transforming the size of the image.

I could send a PR to update the comments in enabledElements.js

**Comment by dannyrb** [comment]
@wwwillchen I'm more than happy to review and merge any PRs that help add clarity ^_^
